packages/edit-navigation/src/components/header/save-button.js

```diff
@@ -19,6 +23,7 @@ export default function SaveButton( { navigationPost } ) {
 			isPrimary
 			onClick={ () => {
 				saveNavigationPost( navigationPost );
+				saveMenuName();
```

Comment:
How are all these changes involving menu name related to menu locations?

Reply:
@draganescu It does say in the PR description:

> Branched from ( #29012 )

But I think the base branch in github should be set to `add/navigation-menu-name-editor` so that all of those changes don't appear in this PR. Github will automatically change the base to `trunk` when that branch is merged.

Reply:
I went ahead and did that.

Reply:
Hmm, this change is still here, but it was part of that other PR. I'll do some tidy up before merge.

Reply:
conflicts resolved;)

Reply:
🤔 I think this change is still from #29012. The menu location change shouldn't require any changes to saving.